src/Tools/dotnet-monitor/DiagnosticPortOptions.cs

```diff
@@ -10,13 +14,20 @@ namespace Microsoft.Diagnostics.Tools.Monitor
 {
     public class DiagnosticPortOptions
     {
+        [Description(@"In Connect mode, dotnet-monitor connects to the application for diagnostics. In Listen
```

Comment:
nit: I think "Connect" and "Listen" should be surrounded by single quotes. There should be a comma after "In Listen mode". And end the sentence with a period.